keras_model_specs/model_spec.py

```diff
@@ -33,13 +33,16 @@ def mean_subtraction(x, args=None):
 SPEC_FIELDS = ['name', 'klass', 'target_size', 'preprocess_func', 'preprocess_args']
 
 
+with open(os.path.join(os.path.split(__file__)[0], 'model_specs.json')) as file:
```

Comment:
How about lazy loading this in `ModelSpec` so it's not opening/reading the file on every `import`?